app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java

```diff
@@ -50,6 +50,7 @@
 
     private View rootView;
 
+    //TODO: What is this doing? What is special about these entries?
     public static String options = "Available|In use|Almost done|End of cycle";
```

Comment:
Fixed it. Its the filter options. I pulled them out, but it still works the same.